pages/common/pbmtoatk.md

```diff
@@ -0,0 +1,9 @@
+# pbmtoatk
+
+> Convert a PBM image into an Andrew Toolkit raster object.
```

Comment:
```suggestion
> Convert a PBM image to an Andrew Toolkit raster object.
```
What do you think of always use "convert ... to ..." instead of "convert ... into ..." in these cases?